scripts/leapfrog/post_leap.sh

```diff
@@ -29,6 +29,8 @@ if [[ ! -f "${UPGRADE_LEAP_MARKER_FOLDER}/deploy-rpc.complete" ]]; then
     # TODO(remove the following hack to restart the neutron agents, when fixed upstream)
     ansible -m shell -a "restart neutron-linuxbridge-agent" nova_compute -i /opt/rpc-openstack/openstack-ansible/playbooks/inventory/dynamic_inventory.py
     openstack-ansible ${RPCO_DEFAULT_FOLDER}/scripts/leapfrog/playbooks/remove-old-agents-from-maas.yml
+    # TO-DO(fran0021) exclude Elasticsearch, upgrade it after leap
```

Comment:
@Frank-ZhangXin @antonym @cloudnull This will also effect K2N leapfrogs should we find ourselves wanting to test/bump future ones to say r14.5 or so after this weekend and the next few upgrades right?

Is there a way to make this L+ only?

Reply:
The pre and post parts in rpc-o of the leapfrog code don't currently have version detection.  We might see if osa_ops drops the leapfrog from variable down anywhere and if not, add it so that we can then key off it in the post scripts.

Reply:
IT may also be that @cloudnull rcbops/rpc-upgrades tackles this via getting these changes there, and using that tool for L2N?

Reply:
That's ultimately where those changes would live but as it is now, the rpc-upgrades/leapfrog scripts in rpc-o only contain the wrapper around the openstack-ansible-ops leapfrog logic.  I'm not sure if that's something we want to fork or leave upstream.  Either way, it looks like the changes need to happen at the end before we run the rpc playbooks.  

We can probably load up a variable there that specifies what the original version was that was upgraded and just set a if != kilo, then run.